solutions/swb-reference/integration-tests/support/complex/paabHelper.ts

```diff
@@ -41,20 +47,18 @@ export class PaabHelper {
       accountId: this._setup.getSettings().get('defaultHostingAccountId'),
       description: 'Cost Center for integ test'
     });
-
-    // create two projects
-    const projectNames: string[] = ['Project1', 'Project2', 'Project3'];
     const projectIds: string[] = [];
 
-    for (const projectName of projectNames) {
-      // must follow Array order
+    for (let i = 1; i <= this._numberOfProjects; i++) {
+      const projectName = `Project${i}`;
       const projectResponse = await adminSession.resources.projects.create({
         name: this._randomTextGenerator.getFakeText(projectName),
         description: `${projectName} for integ tests`,
         costCenterId: costCenter.id
       });
       projectIds.push(projectResponse.data.id);
     }
+
     const [project1Id, project2Id, project3Id] = projectIds;
```

Comment:
What's the expected behavior if `_numberOfProjects = 1`? Would `project2Id` and `project3Id` be `undefined`?

Reply:
Yup, they get set to undefined.